go/signedexchange/cbor/encoder.go

```diff
@@ -0,0 +1,233 @@
+package cbor
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"sort"
+	"unicode/utf8"
+)
+
+var (
+	ErrInvalidUTF8 = errors.New("Cannot encode invalid UTF-8.")
+)
+
+type Encoder struct {
+	w io.Writer
```

Comment:
I assume you gave `w` a name to avoid exporting it?

Reply:
Right.